src/notify.ts

```diff
@@ -0,0 +1,19 @@
+import { Notification, NotificationConstructorOptions } from 'electron'; // tslint:disable-line no-implicit-dependencies
+import settings from 'electron-settings';
+
+const notify = (
+  options: NotificationConstructorOptions,
+  clickHandler: () => void
```

Comment:
Nit: can we change that to an options object? When I saw this code being used the callback wasn't really clear to me.

```
notify({}, {onItemClick: () => { ... })
```

would make it easier to grasp from the other side. :)